### PR TITLE
CTED-446 forcing back calendar node on courses page

### PR DESCRIPTION
**Pull request by kurvinh**
adding back calendar node on courses page

### CODE REVIEW

File: lib.php [hunk 1]
```diff
@@ -625,23 +625,20 @@ function local_boostnavigation_extend_navigation(global_navigation $navigation)
...
-            $coursehomenode->isactive = false;
-        }
+    $calendarnode = navigation_node::create(get_string('calendar', 'calendar'),
+        new moodle_url('/calendar/view.php', array('course' => $COURSE->id, 'view' => 'month')),
+        global_navigation::TYPE_CUSTOM,
+        null,
+        'calendar',
+        new pix_icon('i/calendar', '')
+    );
+    $coursehomenode->add_node($calendarnode, 'localboostnavigationcoursesections');
```
Comment (JJSwanevelder) [inline lib.php:635]: Do you think a check is needed to see if a calendar-node already exists?

**Review by JJSwanevelder — CHANGES_REQUESTED** [review]
Hi @kurvinh see some questions please

File: lib.php (hunk 1 above)
Comment (kurvinh) [reply lib.php:635]: I tried, it's not the issue the current fix removes it from the courses page and LT wants it back now. the original calendar node is hidden via CSS, so would need to remove the previous calendar node on course pages only.

Comment (JJSwanevelder) [reply lib.php:635]: As long as we ensure that there is not a chance for 2 calendar nodes then I am happy.

**Review by JJSwanevelder — APPROVED** [review]
👍

**Pull request merged**